--- a/net/minecraft/command/impl/CloneCommand.java
+++ b/net/minecraft/command/impl/CloneCommand.java
@@ -1,5 +1,6 @@
 package net.minecraft.command.impl;
 
+import com.example.mod.FillLimitCommand;
 import com.google.common.collect.Lists;
 import com.mojang.brigadier.CommandDispatcher;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
@@ -108,9 +109,9 @@
         {
             int i = mutableboundingbox.getXSize() * mutableboundingbox.getYSize() * mutableboundingbox.getZSize();
 
-            if (i > 32768)
+            if (i > FillLimitCommand.fillLimit)
             {
-                throw CLONE_TOO_BIG_EXCEPTION.create(32768, i);
+                throw CLONE_TOO_BIG_EXCEPTION.create(FillLimitCommand.fillLimit, i);
             }
             else
             {
