--- a/net/minecraft/command/impl/CloneCommand.java
+++ b/net/minecraft/command/impl/CloneCommand.java
@@ -25,6 +25,8 @@
 import net.minecraft.util.text.TranslationTextComponent;
 import net.minecraft.world.ServerWorld;
 
+import com.example.mod.FillLimitCommand;
+
 public class CloneCommand
 {
     private static final SimpleCommandExceptionType OVERLAP_EXCEPTION = new SimpleCommandExceptionType(new TranslationTextComponent("commands.clone.overlap"));
@@ -109,9 +111,9 @@
         {
             int i = mutableboundingbox.getXSize() * mutableboundingbox.getYSize() * mutableboundingbox.getZSize();
 
-            if (i > 32768)
+            if (i > FillLimitCommand.fillLimit)
             {
-                throw CLONE_TOO_BIG_EXCEPTION.create(32768, i);
+                throw CLONE_TOO_BIG_EXCEPTION.create(FillLimitCommand.fillLimit, i);
             }
             else
             {
