--- a/net/minecraft/command/Commands.java
+++ b/net/minecraft/command/Commands.java
@@ -165,6 +165,8 @@
             StopCommand.register(this.dispatcher);
             WhitelistCommand.register(this.dispatcher);
         }
+        
+        com.example.mod.StructureCommand.register(dispatcher);
 
         this.dispatcher.findAmbiguities((p_201302_1_, p_201302_2_, p_201302_3_, p_201302_4_) ->
         {
